examples/src/CustomVariableProvider.php

```diff
@@ -37,7 +37,7 @@ class CustomVariableProvider extends StandardVariableProvider implements Variabl
     public function getByPath($path)
     {
         if ($path === 'random') {
-            return 'random' . sha1(rand(0, 999999999));
+            return 'random' . hash('xxh3', (string)rand(0, 999999999));
```

Comment:
Why do you need to hash this at all?

Reply:
unsure. it's example code - didn't wanted to bother with this much at this point.

Reply:
Oh, right, I didn't see it's example code.